ddtrace/internal/ci_visibility/coverage.py

```diff
@@ -16,12 +18,18 @@
 from ddtrace.internal.ci_visibility.telemetry.coverage import record_code_coverage_finished
 from ddtrace.internal.ci_visibility.telemetry.coverage import record_code_coverage_started
 from ddtrace.internal.ci_visibility.utils import get_relative_or_absolute_path_for_path
+from ddtrace.internal.coverage.code import ModuleCodeCollector
 from ddtrace.internal.logger import get_logger
+from ddtrace.internal.utils.formats import asbool
 
 
 log = get_logger(__name__)
 _global_relative_file_paths_for_cov: Dict[str, Dict[str, str]] = {}
 
+# This feature-flags experimental collection of code coverage via our internal ModuleCodeCollector.
+# It is disabled by default because it is not production-ready.
+USE_DD_COVERAGE = asbool(os.environ.get("_DD_USE_INTERNAL_COVERAGE", "false"))
```

Comment:
If we expect this feature to have a set of configuration env vars perhaps we can create a sub-class of `envier.En` in `ddtrace/settings`

Reply:
I don't expect it to, at the moment. This is only a feature flag used for internal testing.

I suppose there may be some set of releases where we want to provide the ability to switch between our internal coverage and the current solution, but I was planning on addressing that at the point where our internal coverage collector becomes the default for ITR.